### PR TITLE
Added grpc_call_stats to the call stack

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by dgquintas** [comment]
It remains to be seen how to actually populate the stats.

File: src/core/lib/channel/channel_stack.h [hunk 1]
```diff
@@ -67,6 +69,13 @@ typedef struct {
   grpc_call_context_element *context;
 } grpc_call_element_args;
 
+typedef struct {
+  uint64_t bytes_in;
+  uint64_t bytes_out;
+  gpr_timespec latency; /* TODO(dgq): per op? */
```
Comment (ctiller) [inline src/core/lib/channel/channel_stack.h]: From call creation to enqueuing the status received op, I think.

File: src/core/lib/channel/channel_stack.h [hunk 2]
```diff
@@ -67,6 +69,13 @@ typedef struct {
   grpc_call_context_element *context;
 } grpc_call_element_args;
 
+typedef struct {
+  uint64_t bytes_in;
```
Comment (ctiller) [inline src/core/lib/channel/channel_stack.h]: These two fields should be exactly grpc_transport_stream_stats (recommend including that struct within this one).

**Comment by dgquintas** [comment]
Done. PTAL.

**Comment by dgquintas** [comment]
superseded by #5901

**Pull request closed without merging**